### ISSUE TITLE
[Enhance] MultiOutput

### ISSUE THREAD

**Issue by karabelaselias** [stelong/GPErks#21]
How about taking advantage of the multioutput GPs in gpytorch? This would make a nice addition for GPerks.

**Comment by stelong** [comment]
Yes, this is likely to be one of the first new features we will work on integrating for the new year 2022 release. Stay tuned!
